Source/Charts/Charts/PieChartView.swift

```diff
@@ -639,4 +642,28 @@ open class PieChartView: PieRadarChartViewBase
             }
         }
     }
+    
+    /// The offset value that is used for shifting the data label outside of the center of chart.
+    /// **default**: 0.0
+    open var dataLabelOffset: CGFloat
```

Comment:
and we should mention min/max in API documentation

Reply:
The min will be 0.0 and max will be 1.0